mappings/net/minecraft/util/math/BlockPos.mapping

```diff
@@ -6,7 +6,10 @@ CLASS net/minecraft/class_2338 net/minecraft/util/math/BlockPos
 	COMMENT <p>A block position may be mutable; hence, when using block positions
 	COMMENT obtained from other places as map keys, etc., you should call {@link
 	COMMENT #toImmutable()} to obtain an immutable block position.
-	FIELD field_10973 BITS_Z J
+	FIELD field_0_5760 BIT_SHIFT_Y I
+	FIELD field_0_5761 BIT_SHIFT_X I
+	FIELD field_0_5763 BITS_Y J
+	FIELD field_10973 BITS_X J
```

Comment:
In which version did you map this? I know for sure it's supposed to be `BITS_Z` for 1.14-Snapshots+. I hope it's not a mismatch on intermediary side.